packages/node/package.json

```diff
@@ -29,6 +29,7 @@
     }
   },
   "dependencies": {
+    "@edge-runtime/feature-detector": "1.0.0",
```

Comment:
I think these should be devDependencies since ncc will bundle it, right?

Reply:
Good point, thank you!